### PR TITLE
Add CSharp FastMarching example.

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

**Comment by dave3d** [comment]
I've built and run this example, and it works for me.

**Comment by blowekamp** [comment]
What specific compiler/version did you use?

**Comment by dave3d** [comment]
Visual Studio 2017.

Oh, and the calls to Show should probably check for the SITK_NOSHOW environment variable so Fiji doesn't get launched when you run all the tests.

File: Examples/FastMarchingSegmentation/FastMarchingSegmentation.cs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            //  feature-preserving diffusion.  We create a filter and set the
+            //  appropriate parameters.
+
+            CurvatureAnisotropicDiffusionImageFilter smoothing = new CurvatureAnisotropicDiffusionImageFilter();
+            smoothing.SetTimeStep(0.125);
+            smoothing.SetNumberOfIterations(5);
+            smoothing.SetConductanceParameter(9.0);
+            SitkImage smoothingOutput = smoothing.Execute(inputImage);
+
+            SimpleITK.Show(smoothingOutput, "smoothed");
```
Comment (blowekamp) [inline Examples/FastMarchingSegmentation/FastMarchingSegmentation.cs]: Remove this and other show commands below.

**Comment by blowekamp** [comment]
@pettaa123 Do you have any insight it the the C# compilation [errors](https://open.cdash.org/viewBuildError.php?buildid=5873039) encounter on the CI builds:
```
d:\a\1\s\Examples\FastMarchingSegmentation\FastMarchingSegmentation.cs(84,58): error CS1002: ; expected [D:\a\1\s-build\SimpleITK-build\Examples\FastMarchingSegmentation\CSharpFastMarchingSegmentation.vcxproj] [D:\a\1\s-build\SimpleITK.vcxproj]
```

**Comment by bradking** [comment]
Do: check

**Comment by pettaa123** [comment]
`Convert.ToDouble("-0..5")` returns 5. Instead I would use `double.Parse(args[4], CultureInfo.InvariantCulture)`

**Comment by pettaa123** [comment]
https://github.com/SimpleITK/SimpleITK/pull/744 i changed the way of value assignment of vectoruint32. Now it passes.

**Comment by blowekamp** [comment]
AWSOME! 🎆 

Thank you so much for figuring this out!

I'm closing this as #744 replaces this PR>

**Pull request closed without merging**